### PR TITLE
[feat/aut998] - colouring logs

### PR DESCRIPTION
**Pull request by ghost**
Colouring logs in SmartPeak based on severity.

image

### CODE REVIEW

**Review by bertrandboudaud — CHANGES_REQUESTED** [review]
Is it possible to have some minor changes (customization):

- Also activate color for the console (it's just to use plog::ColorconsoleAppender instead of ConsoleAppender in the GUI)
- Then use the same color in your code as the one used by plog to have consistency.

image

File: src/smartpeak/include/SmartPeak/ui/GuiAppender.h [hunk 1]
```diff
@@ -32,12 +32,15 @@ namespace SmartPeak
   class GuiAppender : public plog::IAppender
   {
   public:
+    typedef std::pair<plog::Severity, plog::util::nstring> GuiAppenderRecord;
+    
     void write(const plog::Record& record) override;
 
     std::vector<plog::util::nstring> getMessageList(plog::Severity severity) const;
```
Comment (bertrandboudaud) [inline src/smartpeak/include/SmartPeak/ui/GuiAppender.h]: Is getMessageList still useful?

**Comment by ghost** [comment]
I've added the coloured terminal for windows.

**Review by bertrandboudaud — APPROVED** [review]

File: src/smartpeak/include/SmartPeak/ui/GuiAppender.h (hunk 1 above)
Comment (ghost) [reply src/smartpeak/include/SmartPeak/ui/GuiAppender.h]: no, it's been removed

**Review by dmccloskey — APPROVED** [review]

**Pull request merged**